content/soarian-financials/stu3/individuals/related-person.md

```diff
@@ -0,0 +1,106 @@
+---
```

Comment:
I am not going to look at this file at this time, apply all comments left on patient.md here